app/controllers/charges_controller.rb

```diff
@@ -0,0 +1,43 @@
+class ChargesController < ApplicationController
+  def create
+    total = cart.total_price
+    total_in_cents = total * 100
+
+    # Amount in cents
+    @amount = total_in_cents.to_i
+
+    customer = Stripe::Customer.create(
+      email: current_user.email,
+      card:  params[:stripeToken]
+    )
+
+    charge = Stripe::Charge.create(
+      customer:    customer.id,
+      amount:      @amount,
+      description: Order.last.id + 1,
+      currency:    'usd'
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.